app/assets/stylesheets/card.scss

```diff
@@ -0,0 +1,57 @@
+.card {
```

Comment:
I didn't write this intending it to be shareable across apps, but it might be able to. For now, keeping it feeder only.